File: Libreria.view/src/app/book-module/entities/book.ts
import { Ieditorial } from "./Ieditorial";
import { Ibook } from "./Ibook";
import { editorial } from "./editorial";

export class book implements Ibook{
    constructor(){}
    bookId: number=0;
    title: string='';
    date: Date=new Date();
    spend: number =0;
    sugestedPrice: number=0;
    autor: string='';
    editorialId: number=1;
    editorial: Ieditorial | null=null;

}

File: Libreria.Core/Entities/Book.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Libreria.Core.Entities
{
    public class Book : IBook
    {
        public int BookId{get;set;}

        public string Title{get;set;}

        public DateTime Date{get;set;}

        public decimal Spend{get;set;}

        public decimal SugestedPrice{get;set;}

        public string Autor{get;set;}

        public int EditorialId { get; set; }
        public Editorial Editorial{get;set;}
    }
}
